mindmap/mindmap.py

```diff
@@ -9,6 +9,7 @@
 
```

Comment:
why are we modifying this in this PR?

Reply:
Actually, these changes are not necessary for the PR, I will add them later.

Reply:
Now that I see it..., the Exception class change is necessary, the tests were failing the other way.

Reply:
No prob if it works the same